doc/STYLE_GUIDE.md

```diff
@@ -13,13 +13,9 @@
 * American English spelling is preferred. "Capitalize" vs. "Capitalise",
   "color" vs. "colour", etc.
 * Use [serial commas][].
-* Generally avoid personal pronouns in reference documentation ("I", "you",
-  "we").
-  * Pronouns are acceptable in more colloquial documentation, like guides.
-  * Use gender-neutral pronouns and mass nouns. Non-comprehensive
-    examples:
-    * OK: "they", "their", "them", "folks", "people", "developers", "cats"
-    * NOT OK: "his", "hers", "him", "her", "guys", "dudes"
+* Use gender-neutral pronouns and mass nouns. Non-comprehensive examples:
+  * OK: "they", "their", "them", "folks", "people", "developers", "cats"
```

Comment:
How about we remove "cats" while we are here? I know, it is supposed to be funny, but it doesn't seem appropriate.

Reply:
I believe "cats" is slang as in "Those Node developers are hip cats."
It's the counter to "dudes".